test/e2e/tests/request-queuing/multiple-networks-dapps-txs.spec.js

```diff
@@ -146,14 +146,7 @@ describe('Request Queuing for Multiple Dapps and Txs on different networks.', fu
         await driver.switchToWindowWithTitle(
           WINDOW_TITLES.ExtensionInFullScreenView,
         );
-
-        // TODO: Reload fix to have the confirmations show
-        await driver.executeScript(`window.location.reload()`);
-
-        // Second Switch Network
-        await driver.switchToWindowWithTitle(
-          WINDOW_TITLES.ExtensionInFullScreenView,
-        );
```

Comment:
- we don't reload the browser anymore, and instead go to the activity tap
- we already switched to the Extension window, so we can remove this step